src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -698,7 +698,7 @@ TWO_FACTOR_CALL_GATEWAY = 'corehq.apps.hqwebapp.two_factor_gateways.Gateway'
 ENFORCE_TWO_FACTOR_FOR_SUPERUSERS = {{ localsettings.ENFORCE_TWO_FACTOR_FOR_SUPERUSERS }}
 {% endif %}
 
-{% if localsettings.get('CUSTOM_LANDING_TEMPLATE') and isinstance(localsettings.CUSTOM_LANDING_TEMPLATE, str)%}
+{% if localsettings.get('CUSTOM_LANDING_TEMPLATE') and localsettings.CUSTOM_LANDING_TEMPLATE is not mapping %}
```

Comment:
I've never seen this before. Is it a Jinja2 template thing?

Reply:
Yeah. `isinstance` doesn't actually work in this context http://jinja.pocoo.org/docs/2.10/templates/#mapping